src/thrift/parquet.thrift

```diff
@@ -191,6 +191,30 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+enum BloomFilterStrategy{
+  /**
+   * See "Less Hashing, Same Performance: Building a Better Bloom Filter" by Adam Kirsch and
+   * Michael Mitzenmacher. The paper argues that this trick doesn't significantly deteriorate the
+   * performance of a Bloom filter (yet only needs two 32bit hash functions).
+   */
+  MURMUR128_32 = 0;
+
+  /**
+   * This strategy uses all 128 bits of {@link Hashing#murmur3_128} when hashing. It looks
```

Comment:
This should also be more clear and describe exactly what happens to calculate the k bit positions. It is good to describe how it differs from MURMUR128_32, but that should be to clarify rather than how this is defined.